microsoft-edge/webview2/release-notes.md

```diff
@@ -104,6 +104,78 @@ For full API compatibility, this version of the WebView2 SDK requires Microsoft
 <!-- end of top matter -->
 
 
+<!-- ====================================================================== -->
+## 1.0.2045.28
+
+Release Date: September 18, 2023<!-- todo: finalize -->
+
+[NuGet package for WebView2 SDK 1.0.2045.28](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2045.28)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 117.0.2045.28 or higher.
+
+#### General
+
+###### Bug fixes
+
+* Disabled the Mouse Gesture feature by default.  (Runtime-only)  ([Issue #3737](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3737))
+* Fixed a bug where mouse wheel scrolling was intermittently broken for visual hosting.  (Runtime-only)
+* Fixed a bug where downloading APK files in WebView2 crashes the WV2<!--todo: spell out as "WebView2"?  lowercase it?--> browser process.  (Runtime-only)  ([Issue #3569](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3569))
+
+<!-- end of Sep 2023 Release SDK -->
+
+
+<!-- ====================================================================== -->
+## 1.0.2098-prerelease
+
+Release Date: September 18, 2023<!-- todo: finalize -->
+
+[NuGet package for WebView2 SDK 1.0.2098-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2098-prerelease)
+
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 119.0.2098.0 or higher.
+
+#### General
+

```

Comment:
```suggestion
#### WinRT
```
headings structural design: should this RT-specific info be in a WinRT section at same level as General heading, similar to the headings breakout in early release h2 sections?

Reply:
tried a variant instead, for headings structure